js/adapt-languagePicker.js

```diff
@@ -38,6 +38,18 @@ define([
    * If it was, load it. If it wasn't, show the language picker
    */
   function onOfflineStorageReady() {
+    var search = window.location.search.substring(1);
+    var queries = search.split('&');
+
+    queries.forEach(function (query) {
+      var pair = query.split('=');
+
+      if (decodeURIComponent(pair[0]) !== 'lang') return;
+
+      var language = decodeURIComponent(pair[1]);
```

Comment:
I feel like this would be slightly neater if split out into a separate function, so you could something like:
```js
const language = getLanguageFromURL();
if (language) Adapt.offlineStorage.set('lang', language);
```

I'm also thinking that maybe that `getLanguageFromURL` would be useful in the the core code as then it would allow us to take advantage of this even if this plugin isn't being used... and then you could have something like this in configModel.js:
![image](https://user-images.githubusercontent.com/1249597/111627557-15ed4b00-87e7-11eb-93d7-fc0d1d374877.png)

Specifically I'm just thinking ahead to adding multilanguage support to the AAT - this could be useful in allowing the preview to automatically open the course at the language being worked on, rather than the author having to select it in the language picker every time. 

@tomgreenfield / @oliverfoster / @taylortom  would be useful to have your thoughts on this...

Reply:
Yeh I think that could be really useful

Reply:
In fact, we could go as far as expanding it to
`getParamFromURL('lang')` and allow for future parsing of params

Reply:
https://github.com/adaptlearning/adapt_framework/pull/3086

Reply:
yes, indeed there's [some code in spoor](https://github.com/adaptlearning/adapt-contrib-spoor/blob/16a39328cbbeaef310da3aac6f04f857133c2ca0/js/adapt-stateful-session.js#L26-L29) that could make use of this